demo/Demo.vue

```diff
@@ -1,31 +1,35 @@
 <template>
   <div class="demo">
-    <button @click="trySave">Submit</button>
+    <button @click="save">Submit</button>
     <editor
-      autofocus
-      holder-id="codex-editor"
+      :autofocus="true"
```

Comment:
`:autofocus="true"` is same as `autofocus`. but this is more clear to understand. thanks.